metaworld/envs/mujoco/sawyer_xyz/v2/sawyer_bin_picking_v2.py

```diff
@@ -140,22 +140,46 @@ def compute_reward(self, action, obs):
             sigmoid='long_tail',
         )
 
+        threshold = 0.05
+        radii = [
+            np.linalg.norm(hand[:2] - self.obj_init_pos[:2]),
+            np.linalg.norm(hand[:2] - self._target_pos[:2])
+        ]
+        # floor is a *pair* of 3D funnels centered on (1) the object's initial
```

Comment:
this is an interesting technique. 

Can you draw out what you mean? 
I think I understand but the clarification would be nice.
@haydenshively